dashboard/client/src/type/job.d.ts

```diff
@@ -70,6 +70,7 @@ export type UnifiedJob = {
   metadata: { [key: string]: string } | null;
   runtime_env: { [key: string]: string } | null;
   driver_info: DriverInfo | null;
+  driver_agent_http_address: string | null;
```

Comment:
nit:
```suggestion
  driver_agent_http_address: string?;
```

VS code shows this as shorthand for `string | null`, although maybe it's not the case and type? includes undefined too which is why the other fields are defined like this